app/Http/Controllers/Api/V1/UserDataAccessApplicationController.php

```diff
@@ -1189,8 +1172,8 @@ public function destroyFile(DeleteUserDataAccessApplicationFile $request, int $u
     /**
      * @OA\Delete(
      *      path="/api/v1/users/{userId}/dar/applications/{id}",
-     *      summary="Delete a user's DAR application",
-     *      description="Delete a user's DAR application",
+     *      summary="Delete a users DAR application",
+     *      description="Delete a users DAR application",
```

Comment:
Keep the apostrophe unless it's causing swagger issues

Reply:
Sadly, it was causing swagger issues.